eclipse-collections-code-generator-maven-plugin/src/main/java/org/eclipse/collections/codegenerator/maven/GenerateMojo.java

```diff
@@ -63,27 +63,23 @@ public void execute() throws MojoExecutionException
         List<URL> urls = Arrays.asList(((URLClassLoader) GenerateMojo.class.getClassLoader()).getURLs());
 
         boolean[] error = new boolean[1];
-        EclipseCollectionsCodeGenerator.ErrorListener errorListener = new EclipseCollectionsCodeGenerator.ErrorListener()
-        {
-            public void error(String string)
-            {
-                GenerateMojo.this.getLog().error(string);
-                error[0] = true;
-            }
+        EclipseCollectionsCodeGenerator.ErrorListener errorListener = string -> {
+            GenerateMojo.this.getLog().error(string);
```

Comment:
There should be a second inspection firing here about the redundant prefix to `this`

Reply:
Fixed.